src/components/05_pages/Content/Content.js

```diff
@@ -76,22 +83,35 @@ class Content extends Component {
     requestContent: PropTypes.func.isRequired,
     requestContentTypes: PropTypes.func.isRequired,
     contentList: PropTypes.arrayOf(PropTypes.object),
+    requestActions: PropTypes.func.isRequired,
+    actionExecute: PropTypes.func.isRequired,
+    actions: PropTypes.arrayOf(PropTypes.object),
     includes: PropTypes.shape({
       'user--user': PropTypes.object,
     }),
   };
   static defaultProps = {
     contentList: [],
     includes: {},
+    actions: [],
   };
   state = {
     contentTypes: [],
     status: null,
+    action: null,
+    checked: {},
```

Comment:
What do you think should happen if user has selected checkboxes and uses pagination to navigate to the next page? The current Drupal UI discards the checked items from previous pages but with the architecture of this application we could still take them into account. 🤔

Reply:
mhhh, there is something in me which would like to introduce a confirm page/model/something for every action, not just dangerous ones like unpublish. When we have this confirm page, keeping the state across multiple pages seems more safe.